packages/payload/src/collections/operations/find.ts

```diff
@@ -176,14 +183,27 @@ export const findOperation = async <TSlug extends CollectionSlug>(
                   in: result.docs.map((doc) => doc.id),
                 },
               },
+              // Query where the lock is newer than the current time minus lock time
+              {
+                updatedAt: {
+                  greater_than: new Date(new Date().getTime() - lockDurationInMilliseconds),
+                },
+              },
             ],
           },
         })
 
+        const now = new Date().getTime()
         const lockedDocs = Array.isArray(lockedDocuments?.docs) ? lockedDocuments.docs : []
 
+        // Filter out stale locks
+        const validLockedDocs = lockedDocs.filter((lock) => {
+          const lastEditedAt = new Date(lock?.updatedAt).getTime()
+          return lastEditedAt + lockDurationInMilliseconds > now
+        })
```

Comment:
Is this needed? It seems that we already queried for lockedDocuments with the where filter updatedAt.